osu.Game/Screens/Select/SongSelect.cs

```diff
@@ -700,6 +716,8 @@ protected override void Dispose(bool isDisposing)
                 music.TrackChanged -= ensureTrackLooping;
 
             modSelectOverlayRegistration?.Dispose();
+
+            beatmaps.OnInvalidated -= workingBeatmapInvalidated;
```

Comment:
`beatmaps` should be null-checked in case of unfortunate disposal before dependency injection.

Reply:
Has been removed.